x/auth/client/testutil/suite.go

```diff
@@ -1789,6 +1790,7 @@ func (s *IntegrationTestSuite) TestAuxToFeeWithTips() {
 					tc.feePayerArgs...,
 				)
 
+				//nolint:gocritic // rewriting this as a switch statement didn't make sense.
```

Comment:
this is totally what I mean about codecov

Reply:
eg: 

* comments aren't new code lacking test coverage
* nolint statements aren't new code lacking test coverage